qlib/linux_def.rs

```diff
@@ -3266,6 +3267,31 @@ pub struct LibcStat {
     pub pad: [i64; 3],
 }
 
+#[cfg(target_arch = "aarch64")]
+#[derive(Debug, Default, Copy, Clone)]
+#[repr(C)]
+pub struct LibcStat {
```

Comment:
I can only attest that _struct LibcStat_ matches this [cross-reference](https://chromium.googlesource.com/android_tools/+/20ee6d20/ndk/platforms/android-21/arch-arm64/usr/include/sys/stat.h#40). Probably @SchwarzStein and @shrik3 have more insight.

Reply:
it's maybe better to check against the [linux header defs](https://elixir.bootlin.com/linux/v6.6/source/arch/arm64/include/asm/stat.h#L26) for this.

Reply:
Actually, I followed the definition in the libc crate in rust  https://github.com/rust-lang/libc/blob/main/src/unix/linux_like/linux/gnu/b64/aarch64/mod.rs#L53 

> it's maybe better to check against the [linux header defs](https://elixir.bootlin.com/linux/v6.6/source/arch/arm64/include/asm/stat.h#L26) for this.

It seems the definition in the link is for the compat and it is quite different from the 64 bits. @shrik3

Reply:
JFI, the reference to [generic `struct stat` in linux](https://elixir.bootlin.com/linux/v6.6/source/include/uapi/asm-generic/stat.h#L24). If my eyes don't lie, it matches the one from [Rust](https://github.com/rust-lang/libc/blob/main/src/unix/linux_like/linux/gnu/b64/aarch64/mod.rs#L53) that @CharlyYu refers to.

Reply:
@shrik3 If it is ok for you I will merge this pr

Reply:
> @shrik3 If it is ok for you I will merge this pr

I'm ok with this.